### PR TITLE
Top contributors improvements

### PR DESCRIPTION
**Pull request by mathjazz**
Top contributors order is broken, this fixes it and also adds notice on excluding unchanged translations.

@Osmose, r?

### CODE REVIEW

File: pontoon/base/templates/users.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
       <tr>
         <th>Rank</th>
         <th>Contributor</th>
-        <th>Translations</th>
+        <th>Translations <span>* without unchanged</span></th>
```
Comment (Osmose) [inline pontoon/base/templates/users.html]: `sup` would probably be a better tag to use here. 

Also I think adding something like a footer with a complete sentence like "Translation counts do not include unchanged translations." would be better, maybe with a `sup` tag next to the header that links to the footnote.

File: pontoon/base/static/css/users.css [hunk 2]
```diff
@@ -14,6 +14,11 @@ th {
   text-transform: uppercase;
 }
 
+th span {
```
Comment (Osmose) [inline pontoon/base/static/css/users.css]: Font size should probably be smaller too.

**Comment by Osmose** [comment]
Could use some tests to make sure your filtering works as expected.

r+wc, nice work!

**Comment by mathjazz** [comment]
@Osmose r?

I think I forgot to ask for it last week.

File: pontoon/base/templates/users.html [hunk 3]
```diff
@@ -47,6 +47,11 @@
       </tr>
     {% endfor %}
     </tbody>
+    <tfoot>
+      <tr>
+        <td colspan="3"><sup id="fn-translations">Translation counts do not include unchanged translations.<a href="#ref-translations" title="Jump back on top.">↩</a></sup></td>
```
Comment (Osmose) [inline pontoon/base/templates/users.html]: "to top" is more appropriate than "on top" here.

**Comment by Osmose** [comment]
r+, looks great! Even better than I imagined. :D

**Pull request merged**